### PR TITLE
[RDY] Add introductory text to intro screen.

### PR DESCRIPTION
**Pull request by fwalch**
This is an attempt to make the intro screen more helpful. It is the first thing new users see when they start Neovim, and it doesn't really help in addressing some of the problems they might have (e.g. https://github.com/neovim/homebrew-neovim/issues/18#issuecomment-71333806). I tried to do something similar with #1412, but I guess that change was so small that Vim users might not even have noticed it. This should be more visible :wink:

I just added the text from https://github.com/neovim/neovim/issues/1692#issuecomment-69206110 for now, and would like to get your input on this :-)

gui

Note that the text mentions some features we don't have yet (nvimtutor, support resources [maybe even add a dedicated section at neovim.org/support?]), but maybe this message can serve as a starting point to figure out what should be added. I don't expect this to be merged soon.

From https://github.com/neovim/neovim/issues/1692#issuecomment-69206110:

> nvim-intro would then need to mention things like: vimrc can be reused as nvimrc, you need the python client for the builtin python commands (see :help nvim-python), etc (I guess this is planned anyway).
> 
> There have been multiple issues on Github about why some Vim plugin that uses Python doesn't work etc, so I think we need to point out that additional setup is required for that as obviously as possible.
> 
> With "support resources" I mean things like the Github issue tracker, the mailing list, ... The wording can surely be improved, but I think having a few friendly sentences on startup could already make a difference for beginners.

### CODE REVIEW

**Comment by justinmk** [comment]
The intro screen disappears after a couple seconds

**Comment by fwalch** [comment]
> The intro screen disappears after a couple seconds

It only disappears if you start typing into the buffer or execute a command, if I'm not mistaken.

**Comment by fwalch** [comment]
Revised the text a bit. Refers to community page (neovim/neovim.github.io#91) and vim-tutor-mode.

**Comment by fmoralesc** [comment]
On my end, I plan to make some changes to the vim-tutor-mode syntax this week. Once those are done, I'll propose a PR to add vim-tutor-mode to neovim's runtime and the nvimtutor script.

**Comment by Pyrohh** [comment]
Hmm, any reason this is marked as `waiting`? I guess what I'm really asking is "what is this waiting for?".

**Comment by fwalch** [comment]
I wanted to wait until vim-tutor-mode is in master, as the Text refers to that.

**Comment by Pyrohh** [comment]
Ah, thanks for clarifying.

**Comment by fmoralesc** [comment]
Working on it! ;)

File: src/nvim/version.c [hunk 1]
```diff
@@ -972,6 +972,18 @@ void intro_message(int colon)
...
   static char *(lines[]) = {
+    N_("Welcome to Neovim!"),
+    "",
+    N_("First time using a vi-like editor?"),
+    N_("Type :Vimtutor<Enter> to get started!"),
+    "",
+    N_("Already know your way around Vim?"),
+    N_("See :help nvim for an introduction to Neovim."),
+    "",
+    N_("Can't find what you are looking for?"),
```
Comment (fwalch) [inline src/nvim/version.c]: I'm not satisfied with this one.. What about `Stuck and need help?` or `Still have questions?` or `Want to know more?`.

Comment (Pyrohh) [inline src/nvim/version.c]: I like `Still have questions?`; very concise.

File: src/nvim/version.c [hunk 2]
```diff
@@ -972,6 +972,18 @@ void intro_message(int colon)
   int sponsor;
   char *p;
   static char *(lines[]) = {
+    N_("Welcome to Neovim!"),
+    "",
+    N_("First time using a vi-like editor?"),
+    N_("Type :Vimtutor<Enter> to get started!"),
```
Comment (fmoralesc) [inline src/nvim/version.c]: It's `:VimTutor`.

**Comment by fwalch** [comment]
Updated text to use `:Tutor`, thanks @fmoralesc!

**Comment by fwalch** [comment]
Current screen:

gui

@justinmk 
> The intro screen disappears after a couple seconds

Now I see this behavior as well, but this only happens in nvim, not in vim. Did we change this behavior?

**Comment by fmoralesc** [comment]
> The intro screen disappears after a couple seconds

@fwalch Do you have deoplete installed? I can only reproduce that with it enabled. I suspect it's something it's doing on the `CursosHold` event.

**Comment by fwalch** [comment]
> @fwalch Do you have deoplete installed? I can only reproduce that with it enabled. I suspect it's something it's doing on the CursosHold event.

:+1: That's it.

File: src/nvim/version.c [hunk 3]
```diff
@@ -1041,6 +1041,18 @@ void intro_message(int colon)
   int sponsor;
   char *p;
   static char *(lines[]) = {
+    N_("Welcome to Neovim!"),
```
Comment (justinmk) [inline src/nvim/version.c]: This is too folksy. I think NVIM_VERSION_LONG should remain at the top.

File: src/nvim/version.c [hunk 4]
```diff
@@ -1041,6 +1041,18 @@ void intro_message(int colon)
...
+    "",
+    N_("First time using a vi-like editor?"),
+    N_("Type :Tutor<Enter> to get started!"),
+    "",
+    N_("Already know your way around Vim?"),
+    N_("See :help nvim for an introduction to Neovim."),
+    "",
+    N_("Still have questions?"),
+    N_("Reach out to the Neovim community at neovim.io/community."),
```
Comment (justinmk) [inline src/nvim/version.c]: These lines should go under 1058

**Comment by fwalch** [comment]
Updated.

**Comment by justinmk** [comment]
LGTM

**Comment by fwalch** [comment]
Will merge after #2351 is in.

**Comment by justinmk** [comment]
Merged

**Pull request closed without merging**